frontend/packages/ux-editor/src/hooks/mutations/useDeleteLayoutMutation.ts

```diff
@@ -38,12 +38,7 @@ export const useDeleteLayoutMutation = (org: string, app: string, layoutSetName:
     mutationFn: async (layoutName: string) => {
       let layouts = ObjectUtils.deepCopy(formLayouts);
       delete layouts[layoutName];
-      layouts = await addOrRemoveNavigationButtons(
-        layouts,
-        saveLayout,
-        undefined,
-        formLayoutSettings.receiptLayoutName,
-      );
+      layouts = await addOrRemoveNavigationButtons(layouts, saveLayout, undefined);
```

Comment:
`undefined` er vel standerd for parameter nr. 3, så vi kan nok fjerne den også.
```suggestion
      layouts = await addOrRemoveNavigationButtons(layouts, saveLayout);
```